### PR TITLE
use the os.environ.get shortcut with default arg

### PR DESCRIPTION
**Pull request by Vikka**

### CODE REVIEW

**Comment by Vikka** [comment]
Flake8 issue is solved by this PR: https://github.com/tournesol-app/tournesol/pull/709

File: backend/settings/settings.py [hunk 1]
```diff
@@ -16,15 +16,14 @@
 import yaml
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
+from dotenv import load_dotenv
```
Comment (amatissart) [inline backend/settings/settings.py:19]: This import could be moved above the comment about `BASE_DIR`.

File: backend/settings/settings.py [hunk 2]
```diff
@@ -16,15 +16,14 @@
...
 
+load_dotenv()
 
 server_settings = {}
-SETTINGS_FILE = (
-    "SETTINGS_FILE" in os.environ
-    and os.environ["SETTINGS_FILE"]
-    or "/etc/django/settings-tournesol.yaml"
-)
+SETTINGS_FILE = os.getenv("SETTINGS_FILE", "/etc/django/settings-tournesol.yaml")
```
[the remark this answers is not in the document]
Comment (Vikka) [reply backend/settings/settings.py:26]: Exactly. This is a very practical way to handle env variable.

File: backend/settings/settings.py (hunk 1 above)
Comment (Vikka) [reply backend/settings/settings.py]: thanks, updated.

**Review by amatissart — APPROVED** [review]

**Pull request merged**